client/view/index/Index.html

```diff
@@ -1,25 +1,37 @@
 <div class="container">
 
+	{#if message}
```

Comment:
Design change: I actually don't think we should put any of this on the home screen at all - I think that if the user hits the Remove button on the workbook info page, they should stay there on that page, but the "Remove" button should go away and be replaced with an "Undo" button.

It's a better user experience: when they're bounced to a different screen, they have to look for the "undo" functionality among a whole bunch of other pieces of user interface that appeared at the same time.  And if they do decide to undo the removing of the sheet, then they're right back where they were, and can keep interacting with the info screen.